Bylaws.md

```diff
@@ -104,3 +104,42 @@ The purpose of _The Polytechnic_ shall be to keep the members of the Rensselaer
+            
+            1.	_The Polytechnic_ will endorse a candidate based upon a five-sixths vote of quorum.
+            
+            1.	If not one and only one candidate receives an endorsement in the first ballot then the editorial board will again have discussion and vote by secret ballot. If for a second time not one and only one candidate receive an endorsement, then the editorial board will have one last period of discussion and vote by secret ballot a final time. If not one and only one candidate is endorsed at this point, then _The Polytechnic_ shall not endorse a candidate for that position. 
+            
+            1.	Abstentions may only be used for conflicts of interest and count as a vote for a null candidate. Votes for a null candidate are tallied as are those for a real candidate.
+
+            1.	Neither absentee ballots nor voting by proxy are allowed for candidate endorsement meetings.
+
+    1. Endorsement meetings shall be held for the purpose of deciding whether to support or oppose referendas voted on by the student body during Rensselaer Union-sponsored election.
+        1. Meetings during which referenda are considered shall be closed to all except those outlined in clause(iii). Quorum shall consist of two-thirds of the editorial board membership.
+        
+        1. The editor in chief shall call these meetings and decide their location and time.
+        
+        1. Referenda Endorsement Voting Procedure
+            1. For each referendum considered, the meeting participants will discuss the merits of the referendum and then the editorial board will vote by secret ballot whether to support, oppose, or take no stance on the referendum.
+            
+            1. 	_The Polytechnic_ will only support or oppose a referendum based upon a five-sixths vote of quorum in either the affirmative or the negative.
+
+            1. If no stance is endorsed in the first ballot, then the editorial board will again have discussion and vote by secret ballot. If for a second time no stance is endorsed, then the editorial board will have one last period of discussion and vote by secret ballot a final time. If no stance is endorsed at this point, then _The Polytechnic_ shall not support or oppose to the referendum in question.
+
+            1. Abstentions may only be used for conflicts of interest.
+
+            1. Neither absentee ballots nor voting by proxy are allowed for referenda endorsement meetings.
+            
+    1. All editoral board members may participate in endorsement meetings. Other memebers of _The Poly_ may be invited to participate by the disgrssion of the editor in chief. The editorial board may reject the invite by a 2/3 vote.
```

Comment:
"disgrssion" and also i totally disagree with rejecting the invite by a 2/3 vote